templates/includes/records/archive-description.html

```diff
@@ -59,7 +59,7 @@ <h2>Summary of collections</h2>
                     <ul>
                         {% for collection in record.archive_collections.collection_info_list %}
                             {% if collection.count > 0 %}
-                                <li><a href="">{{ collection.display_name }}({{ collection.count }})</a></li>
+                                <li><a href="#heading-{{collection.name}}">{{ collection.display_name }}({{ collection.count }})</a></li>
```

Comment:
@TNA-Allan Any chance of getting a space after the `display_name` here (as suggested previously)?